src/components/Players/PlayerManager.vue

```diff
@@ -279,28 +284,19 @@ export default Vue.extend({
   },
   mounted() {
     document.addEventListener('mousemove', this.handleMouseMove);
+    window.addEventListener('keyup', this.handleKeyPress);
+    window.addEventListener('click', this.handleVideoClick);
```

Comment:
Why not ``@ keyup`` or ``@ click`` in the template?

Reply:
Click bc overlay on the video (and I did not find the right elements after several attempts).

For the keyup I had tried to use that of the Dialog however I encountered some strange behaviors.

After the attempts I made this turned out to be the mix that provided the best experience.

I can try to go through it a second time and double check afterwards.